clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -930,7 +932,12 @@ invariant_checker_t::check_for_pc_discontinuity(
     std::string error_msg = "";
     bool have_branch_target = false;
     addr_t branch_target = 0;
-    const addr_t prev_instr_trace_pc = prev_instr.instr.addr;
+#ifdef UNIX
+    addr_t prev_instr_trace_pc = shard->last_instr_in_cur_context_.instr.addr;
```

Comment:
Add a comment explaining why it needs to be last_instr_in_cur_context_: because we want to check continuity with the last instruction at the same signal nesting depth as the current instr (and not the previous executed instruction which may be from an inner nested signal if a signal just ended).